armi/physics/neutronics/globalFlux/tests/test_globalFluxInterface.py

```diff
@@ -0,0 +1,150 @@
+"""
+Tests for generic global flux interface.
+"""
+import unittest
+
+import numpy
+
+from armi import settings
+from armi.reactor import reactors
+
+from armi.physics.neutronics.globalFlux import globalFluxInterface as gi
```

Comment:
Consider capitalizing, and or renaming GF or GFI. global interface is not super clear to me

Reply:
I'll just remove the rename completely.